client/components/header-cake/back.jsx

```diff
@@ -63,13 +64,10 @@ export default React.createClass( {
 		} );
 
 		return (
-			<a className={ linkClasses } href={ href } onClick={ onClick }>
-				<Gridicon icon="chevron-left" size={ 18 } />
-				{
```

Comment:
Looking at the logic for `hideText` I believe that it only displays the back button text if window width is `>` 480px or if the button text is `<` 8 characters.

So, we need a header cake where the back text is `>` 8 chars. For that, let's use the devdocs since it uses "All App Components" as its back text.

Here's a link:

http://calypso.localhost:3000/devdocs/app-components/comment-button

Here's what the headercake looks like on `wpcalypso`:

![screen shot 2016-05-20 at 4 08 33 pm](https://cloud.githubusercontent.com/assets/1126811/15442255/77b7bd8a-1ea5-11e6-8e8a-f1341a2e7ed9.png)

And `development` after this change:

![screen shot 2016-05-20 at 4 08 54 pm](https://cloud.githubusercontent.com/assets/1126811/15442260/7efd9858-1ea5-11e6-887b-3f47f666a498.png)